.gitignore

```diff
@@ -0,0 +1,2 @@
+Question.txt
+to do.txt
```

Comment:
You need to setup newlines (on save) in your VSCode 

https://cifronomika.ru/blog/git-no-newline-at-end-of-file.html